src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,24 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+    private static final int
+            HUNDRED_MS_PER_SEC = 10,
+            SEC_PER_MIN = 60,
+            FALCON_TICKS = 2048;
+
+    public static double hundredMsToSec(double ms) {
+        return ms * HUNDRED_MS_PER_SEC;
+    }
+
+    public static double hundredMsToMin(double ms) {
+        return hundredMsToSec(ms) * SEC_PER_MIN;
```

Comment:
The calculation is wrong
For example, if I put in 1000ms (ms will be 10) then the function will return 600 instead of 0.01666